model/JobFamily.ts

```diff
@@ -0,0 +1,6 @@
+export interface JobFamily
```

Comment:
Could we rename this to be JobFamilyResponse. We should update the file name to match
```suggestion
export interface JobFamilyResponse
```